compute_runtime/compute_runtime.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Fastly, Inc.
+//
+package compute_runtime
```

Comment:
As an idiomatic thing, underscores aren't used much in Go identifiers.  (Much to my chagrin.)  So we should find a different name for this.  Some options:

* `computeruntime` -- this is a lot and my eyes can't help but see "computer" in there
* `compute` - ok
* `runtime` - there's a stdlib package called this, but the likelihood that some code will import both is small.  Still, maybe best to avoid it
* `compute/runtime` - same issue as the stdlib package.  this would nest a level in the filesystem though.

My ranked voting would be `compute` first and `compute/runtime` second.

Reply:
Oh, interesting. That explains some naming things I've noticed here and there. OK, in that case, I went with `compute`, sort of. I noticed with the "internal" versions, it looks like we are maintaining the underscore and then only removing it for the handy ones we expect folks to use. (Specifically, I note `device_detection` and `secret_store`.) So I left things as `compute_runtime` in `internal`, and then only changed it to `compute` on the "outside".

If those two examples are not two I should've learned from, I'm happy to change the `internal` version too. 
:laughing:

Reply:
You came to the right conclusion.  Underscores in filenames are fine, and the convention.  But two "stronger" conventions are (1) package names should match directory names and (2) identifiers should not have underscores, so these tend to result in directories not having underscores in them.

If you want to dive deeper into this, see https://go.dev/blog/package-names.